docs/source/models.rst

```diff
@@ -5185,7 +5187,11 @@ Deployments can be generated using both the Python API or MLflow CLI. In both ca
   * Copy the ``MLflow tracking URI`` value from the properties section.
 
 * Programmatically, using Azure ML SDK with the method `Workspace.get_mlflow_tracking_uri() <https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.workspace.workspace?view=azure-ml-py#azureml-core-workspace-workspace-get-mlflow-tracking-uri>`_. If you are running inside Azure ML Compute, like for instance a Compute Instance, you can get this value also from the environment variable ``os.environ["MLFLOW_TRACKING_URI"]``.
-* Manually, for a given Subscription ID, Resource Group and Azure ML Workspace, the URI is as follows: ``azureml://eastus.api.azureml.ms/mlflow/v1.0/subscriptions/<SUBSCRIPTION_ID>/resourceGroups/<RESOURCE_GROUP_NAME>/providers/Microsoft.MachineLearningServices/workspaces/<WORKSPACE_NAME>``
+* Manually, for a given Subscription ID, Resource Group and Azure ML Workspace, the URI is as follows: 
+
+    ``azureml://eastus.api.azureml.ms/mlflow/v1.0/subscriptions/``
```

Comment:
Never noticed this before today, but the existing uri path here extends off the edge of the page for a screen in 1080p resolution at 100% zoom.